modules/imaging_browser/jsx/ImagingBrowserDDT.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * Created by dblader on 7/25/17.
+ */
+/* global DynamicDataTable */
+class ImagingBrowserDDT extends DynamicDataTable {
```

Comment:
yeah i know about React's stance on inheritance, however i don't think composition is a viable route here without making substantial changes to both DynamicDataTable and StaticDataTable, since basically what we need here is an override functionality.
i'm not sure if i'm missing some obvious solution here, but i think this is as good as it's going to get within the 17.1 release timeline...